R/utilities-data-combined.R

```diff
@@ -156,7 +156,7 @@ createDataCombinedFromExcel <- function(file, sheet = NULL, dataCombinedNames =
       stop(messages$stopInvalidScenarioName(missingScenarios))
     }
     warning(messages$warningInvalidScenarioName(missingScenarios))
-    dfDataCombined <- dfDataCombined[!(missingScenarios == dfDataCombined$scenario), ]
+    dfDataCombined <- filter(dfDataCombined, dataType == "observed" | !(scenario %in% missingScenarios))
```

Comment:
This should remove all rows where `dataType` is `simulated` (== not observed) and the name of the scenario is one of the `missingScenarios`.

Reply:
> and
you meant "or" right ?. 

Anyway, I would suggest to put the first condition in `( )`, it could prevent some weird things happening in case these condition evolve.